sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/engine/SyncScheduler.kt

```diff
@@ -43,10 +44,12 @@ class RealSyncScheduler @Inject constructor(private val syncStateRepository: Syn
                 IN_PROGRESS -> DISCARD
                 FAIL -> EXECUTE
                 else -> {
-                    val syncTime = Instant.parse(lastSync.timestamp)
-                    val now = Instant.now()
-                    val duration = Duration.between(syncTime, now)
-                    if (duration.toMinutes() > DEBOUNCE_PERIOD_IN_MINUTES) {
+                    val syncTime = OffsetDateTime.parse(lastSync.timestamp)
```

Comment:
Based on a conversation yesterday. This made sense when we were doing one single operation for all features, now that we do sync per feature, I think this should be revisited. For example: FAIL refers to all operations or single feature, one feature failing will mark syncstate as fail or success? will it depend on the response code? etc.

Can you create a class somewhere to don't miss this recent change? I'm ok not working on this now, but we should have it in the roadmap